website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -169,6 +169,8 @@ A `role_based_access_control` block supports the following:
 
 * `azure_active_directory` - (Required) An `azure_active_directory` block. Changing this forces a new resource to be created.
```

Comment:
is this still required? Or can be skipped if one wants just RBAC?

Reply:
good spot - this is now optional